app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -157,13 +188,18 @@ public void onClick(View v) {
                     getChildFragmentManager().beginTransaction().show(transportButtonFragment).commit(); //To be removed after Outside transportation Google API incorporation
                     campusButton.setVisibility(View.VISIBLE);
                     viewSwitchButton.setText("GO INDOORS");
```

Comment:
Store it in Strings.xml and retrieve it from there. Applies to all other instances